index.js

```diff
@@ -16,23 +16,28 @@ export default class FTPTail extends EventEmitter {
       fetchInterval: 0,
       tailLastBytes: 10 * 1000,
       log: false,
+      mode: 'ftp',
       ...options
     };
 
-    // Setup basic-ftp client.
-    this.client = new Client(this.options.ftp.timeout);
-    this.client.ftp.encoding = this.options.ftp.encoding || this.client.ftp.encoding;
-
     // Setup logger.
     if (typeof this.options.log === 'function') {
       this.log = this.options.log;
-      this.client.ftp.log = this.options.log;
     } else if (this.options.log) {
       this.log = console.log;
-      this.client.ftp.log = console.log;
     } else {
       this.log = () => {};
-      this.client.ftp.log = () => {};
+    }
```

Comment:
We could actually get `ssh2-sftp-client`'s logging if we wanted by passing in the `debug` parameter:
https://github.com/theophilusx/ssh2-sftp-client/blob/master/src/index.js#L36